specification/batch/Azure.Batch/models.tsp

```diff
@@ -985,10 +985,10 @@ model BatchPoolResourceStatistics {
   peakDiskGiB: float32;
 
   @doc("The total number of disk read operations across all Compute Nodes in the Pool.")
-  diskReadIOps: int32;
+  diskReadIOps: int64;
```

Comment:
Are these truly `int64`s? Values over 2^53-1 are not interoperable in JSON.

Reply:
Yes, we can confirm that these values are indeed defined as `int64`  (long) in our service codebase. For example:
![diskops](https://github.com/Azure/azure-rest-api-specs/assets/42812319/41e9e09e-6f80-44b9-b0f0-77ec737c863d)